code/game/machinery/bots/mulebot.dm

```diff
@@ -757,7 +757,11 @@ var/global/mulebot_count = 0
 	H.apply_damage(0.5*damage, BRUTE, "l_arm")
 	H.apply_damage(0.5*damage, BRUTE, "r_arm")
 	bloodiness += 4
-	currentBloodColor=bloodcolor // For if species get different blood colors.
```

Comment:
He's saying

 /obj/machinery/bot/mulebot/proc/RunOverCreature(var/mob/living/H,var/bloodcolor)

Why is the mob's species color not being passed as bloodcolor?